rails-buildpack/2.4/Dockerfile

```diff
@@ -0,0 +1,63 @@
+FROM ruby:2.4.5-jessie
```

Comment:
2.4 base image is jessie version because our `rails-base:2.4` is based on jessie. 2.4 EOL is close so I don't want to make a big change